server/test/workflow-execution.test.js

```diff
@@ -46,6 +46,7 @@ describe('Workflow Execution', function() {
         workflowExecutionInfo: null,
         pendingChildren: null,
         pendingActivities: null,
+        pendingDecision: null,
```

Comment:
> updated IDL to latest
IDL now returns new structure, so integration test needs to be updated with new structure to pass.